src/ucs/config/global_opts.c

```diff
@@ -165,6 +166,15 @@ static ucs_config_field_t ucs_global_opts_table[] = {
   "memtrack report is generated and process is terminated.",
   ucs_offsetof(ucs_global_opts_t, memtrack_limit), UCS_CONFIG_TYPE_MEMUNITS},
 
+ {"MEMLOCK_LIMIT", "512m",
+  "Locked memory limit threshold required for proper UCX work.\n"
+  "The default value guarantees the proper work of all dependent UCX parts\n"
+  "in most cases. But in some cases the proper work can be guaranteed\n"
+  "by more or less strict limit threshold.\n"
+  "Please change the limit threshold only if you cannot change \n"
+  "system locked memory limit (ulimit -S -l unlimited).",
+  ucs_offsetof(ucs_global_opts_t, memlock_limit), UCS_CONFIG_TYPE_MEMUNITS},
```

Comment:
I'd rephrase it somehow. For instance:
`The system limit for maximum amount of locked memory must not be smaller than this value.
Otherwise some UCX transports may not function properly. 
Please change this threshold only if you cannot change system locked memory limit (ulimit -S -l unlimited). `

Reply:
also, pls wrap lines (they should be longer)

Reply:
Done.

But shouldn't we mention that this limit value is valid for the most, but not for all the systems. Are we sure that there are no systems when this limit isn't enough for proper work? WDYT about something like:
```
The system limit for maximum amount of locked memory must not be smaller than this value to guarantee the propers UCX work on most systems. Please change this threshold only if you cannot change system locked memory limit (ulimit -S -l unlimited).
```